### ISSUE TITLE
"Message not found in the database" error when opening threads

### ISSUE THREAD

**Issue by georgeowell** [fraction/oasis#201]
I am getting this error every time I open any thread. 
Everything else is working fine. Is anyone else seeing this? 

```
Error: Message not found in the database. You've done nothing wrong. Maybe try again later?
    at /home/user/git/oasis/src/models.js:1121:17
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async thread (/home/user/git/oasis/src/index.js:406:24)
    at async /home/user/git/oasis/src/index.js:412:16
    at async middleware (/home/user/git/oasis/src/index.js:607:5)
    at async /home/user/git/oasis/src/index.js:602:5
    at async /home/user/git/oasis/src/http.js:28:5
    at async /home/user/git/oasis/node_modules/koa-mount/index.js:52:26
```
Latest commit https://github.com/fraction/oasis/commit/f70dae3f0e091765c43a84a19b213f1f0ede8288

**Comment by jedahan** [comment]
There is a PR #200 to fix this, I am testing and will merge soon.

**Comment by georgeowell** [comment]
@jedahan BEST TECH SUPPORT EVER :heart_decoration:

**Comment by jedahan** [comment]
Try updating (`npm --global install github:fraction/oasis`) and see if that fixes it?

**Comment by christianbundy** [comment]
Sorry to cause this bug! It was introduced in #196 and resolved for me in #200. Hope the fix works for you too!

**Comment by georgeowell** [comment]
Solved by https://github.com/fraction/oasis/pull/200
